scripts/upgrade

```diff
@@ -71,19 +71,19 @@ fi
 #=================================================
 
 [[ $YNH_APP_ID == "garradin" ]] \
-    && [[ "$(cat "/opt/$app/templates/.VERSION")" != 1.2.2 ]] \
+    && [[ "$(cat "/$final_path/VERSION")" != 1.2.2 ]] \
     && ynh_die --message "It look like that you have an old Garradin install. You need first upgrade Garradin instance (id: $garradin_migrate_id) and after migrate to Paheko."
 ynh_handle_app_migration --migration_id=garradin --migration_list=garradin_migrations
 
 if [[ $migration_process -eq 1 ]]; then
     # Reload variables
-    dbname=$app
-    db_user=$app
+    #dbname=$app
+    #db_user=$app
 
     # Replace the user
     ynh_system_user_delete --username="$old_app"
-    test getent passwd "$app" &>/dev/null || \
-        useradd -d "$datadir" --system --user-group "$app" --shell /bin/bash || \
+    #test getent passwd "$app" &>/dev/null || \
+        useradd -d "$final_path" --system --user-group "$app" || \
```

Comment:
Maybe worth to use this helper:
https://yunohost.org/fr/packaging_apps_helpers#ynh-system-user-create

You would not have to call `ynh_die` for example, as the case is already handled:
https://github.com/YunoHost/yunohost/blob/394907ff0df218ec926f43c2e2ba006c68cc3f8a/helpers/user#L138

Reply:
So I can commit all these lines ?? just let the script `ynh_handle_app_migration --migration_id=garradin --migration_list=garradin_migrations` ??

Reply:
or delete them ok

Reply:
Ok, I better use the script from install `ynh_system_user_create --username=$app --home_dir="$final_path"`, I a little mistake because I have duplicate the gogs to gitea migration, but it was different case...